packages/shared/src/components/squads/Details.tsx

```diff
@@ -58,6 +68,8 @@ export function SquadDetails({
   children,
   isLoading,
   squad,
+  initialData,
```

Comment:
Just to be safe and so the spread won't break the app.

```suggestion
  initialData = {},
```